tests/system/CMakeLists.txt

```diff
@@ -441,6 +451,17 @@ ENDIF()
                     0 # VMC
                     TRUE)
 
+IF(ENABLE_SOA)
```

Comment:
Hybrid rep are build on top of new spline and new distance table implementation. No support to AoS.